core/unit_test/TestMDRangePolicyConstructors.hpp

```diff
@@ -99,15 +99,37 @@ TEST(TEST_CATEGORY_DEATH, policy_invalid_bounds) {
 
   ::testing::FLAGS_gtest_death_test_style = "threadsafe";
 
-  auto dim = (Policy::inner_direction == Kokkos::Iterate::Right) ? 1 : 0;
+  auto [dim0, dim1] = (Policy::inner_direction == Kokkos::Iterate::Right)
+                          ? std::make_pair(1, 0)
+                          : std::make_pair(0, 1);
+  std::string msg1 =
+      "Kokkos::MDRangePolicy bounds error: The lower bound (100) is greater "
+      "than its upper bound (90) in dimension " +
+      std::to_string(dim0) + ".\n";
```

Comment:
```suggestion
      std::to_string(dim0) + ".";
```

Looking at the CI error output:
```
Death test: { (void)Policy({100, 100}, {90, 90}); }
    Result: died but not with expected error.
  Expected: contains regular expression "Kokkos::MDRangePolicy bounds error: The lower bound (100) is greater than its upper bound (90) in dimension 1.\n"
Actual msg:
[  DEATH   ] Kokkos::MDRangePolicy bounds error: The lower bound (100) is greater than its upper bound (90) in dimension 1.
```
You might not need `\n`.

Reply:
Well we do strict comparison in the non death test code path.
Anyway I tried dropping it here in the definition and appending it back if not doing death tests.